src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/FindOrganisationsByPbaStatusTest.java

```diff
@@ -0,0 +1,105 @@
+import uk.gov.hmcts.reform.professionalapi.util.AuthorizationEnabledIntegrationTest;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.professionalapi.helper.OrganisationFixtures.organisationRequestWithAllFields;
+
+@SuppressWarnings("unchecked")
+public class FindOrganisationsByPbaStatusTest extends AuthorizationEnabledIntegrationTest {
+
+    @Test
+    public void get_request_returns_organisations_with_correct_pba_status() throws JsonProcessingException {
+        createOrganizationsAndEditPbas();
+
+        List<OrganisationsWithPbaStatusResponse> orgPbaResponse = (List<OrganisationsWithPbaStatusResponse>)
+                professionalReferenceDataClient.findOrganisationsByPbaStatus(
+                        PbaStatus.ACCEPTED.toString(), hmctsAdmin, Boolean.FALSE);
+
+        assertThat(orgPbaResponse).isNotNull();
+    }
+
+    @Test
+    public void get_request_returns_no_organisations_for_given_pba_status() throws JsonProcessingException {
+        createOrganizationsAndEditPbas();
+
+        List<OrganisationsWithPbaStatusResponse> orgPbaResponse = (List<OrganisationsWithPbaStatusResponse>)
+                professionalReferenceDataClient.findOrganisationsByPbaStatus(
+                        PbaStatus.REJECTED.toString(), hmctsAdmin, Boolean.FALSE);
+
+        assertThat(orgPbaResponse).isEmpty();
+    }
+
+    @Test
+    public void get_request_returns_400_when_invalid_pba_status() throws JsonProcessingException {
+        createOrganizationsAndEditPbas();
+
+        Map<String, Object> errorResponseMap = (Map<String, Object>) professionalReferenceDataClient
+                .findOrganisationsByPbaStatus("Invalid Status", hmctsAdmin, Boolean.FALSE);
+        ErrorResponse errorResponse = (ErrorResponse) errorResponseMap.get("response_body");
+
+        assertThat(errorResponseMap).isNotNull().containsEntry("http_status", HttpStatus.BAD_REQUEST);
+        assertThat(errorResponse.getErrorMessage()).isEqualTo(ErrorConstants.INVALID_REQUEST.getErrorMessage());
+    }
+
+    @Test
+    public void get_request_returns_403_when_invalid_role() throws JsonProcessingException {
+        createOrganizationsAndEditPbas();
+
+        Map<String, Object> errorResponseMap = (Map<String, Object>) professionalReferenceDataClient
+                .findOrganisationsByPbaStatus(PbaStatus.ACCEPTED.toString(), "invalid role", Boolean.FALSE);
+        ErrorResponse errorResponse = (ErrorResponse) errorResponseMap.get("response_body");
+
+        assertThat(errorResponseMap).isNotNull().containsEntry("http_status", HttpStatus.FORBIDDEN);
+        assertThat(errorResponse.getErrorMessage()).isEqualTo(ErrorConstants.ACCESS_EXCEPTION.getErrorMessage());
+    }
+
+    @Test
+    public void get_request_returns_401_when_invalid_role() throws JsonProcessingException {
```

Comment:
I guess this scenario is not `invalid-role`, rather it is a role that doesn't have permissions. Consider renaming it.